tests/Packetizer/PacketizerTests.cpp

```diff
@@ -2,8 +2,7 @@
 
 #include <cstring>
 
-extern "C"
-{
+extern "C" {
```

Comment:
Change the .clang-format to prevent this change.

Reply:
Done